packages/sdk/src/contracts/classes/edition-drop.ts

```diff
@@ -43,24 +43,21 @@ import { BigNumber, BigNumberish, constants } from "ethers";
  *
  * @public
  */
-export class EditionDrop extends StandardErc1155<DropERC1155> {
-  static contractType = "edition-drop" as const;
-  static contractRoles = ["admin", "minter", "transfer"] as const;
-  static contractAbi = ABI as any;
-  /**
-   * @internal
-   */
-  static schema = DropErc1155ContractSchema;
+export class EditionDropImpl extends StandardErc1155<DropERC1155> {
```

Comment:
kinda wish these were in their own folder but it's fine. `/classes` is getting bloated

Reply:
these are in their own folder (`contracts/classes`)